diff -Naur a/Modules/GNUInstallDirs.cmake b/Modules/GNUInstallDirs.cmake
--- a/Modules/GNUInstallDirs.cmake	2026-08-25 17:10:30.000000000 +0200
+++ b/Modules/GNUInstallDirs.cmake	2026-09-28 22:02:34.404847194 +0200
@@ -467,7 +467,7 @@
       # This host system uses a 'lib64' directory for 64-bit architectures.
       # See https://gitlab.com/x86-psABIs/x86-64-ABI
       if("${CMAKE_SIZEOF_VOID_P}" EQUAL "8")
-        set(${out_var} "lib64")
+        set(${out_var} "lib")
       endif()
     endif()
   endif()
diff -Naur a/Modules/Platform/UnixPaths.cmake b/Modules/Platform/UnixPaths.cmake
--- a/Modules/Platform/UnixPaths.cmake	2026-08-25 17:10:30.000000000 +0200
+++ b/Modules/Platform/UnixPaths.cmake	2026-09-28 22:02:34.405488456 +0200
@@ -78,7 +78,7 @@
   )
 
 list(APPEND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES
-  /lib /lib32 /lib64 /usr/lib /usr/lib32 /usr/lib64
+  /usr/lib
   )
 
 if(CMAKE_SYSROOT_COMPILE)
@@ -116,6 +116,6 @@
 endif()
 
 # Enable use of lib32 and lib64 search path variants by default.
-set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB32_PATHS TRUE)
-set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS TRUE)
-set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIBX32_PATHS TRUE)
+set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB32_PATHS FALSE)
+set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS FALSE)
+set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIBX32_PATHS FALSE)
diff -Naur a/Source/cmExportInstallCMakeConfigGenerator.cxx b/Source/cmExportInstallCMakeConfigGenerator.cxx
--- a/Source/cmExportInstallCMakeConfigGenerator.cxx	2026-08-25 17:10:30.000000000 +0200
+++ b/Source/cmExportInstallCMakeConfigGenerator.cxx	2026-09-28 22:02:34.405986266 +0200
@@ -162,12 +162,7 @@
     os << "# Compute the installation prefix relative to this file.\n"
           "get_filename_component(_IMPORT_PREFIX"
           " \"${CMAKE_CURRENT_LIST_FILE}\" PATH)\n";
-    if (cmHasLiteralPrefix(absDestS, "/lib/") ||
-        cmHasLiteralPrefix(absDestS, "/lib64/") ||
-        cmHasLiteralPrefix(absDestS, "/libx32/") ||
-        cmHasLiteralPrefix(absDestS, "/usr/lib/") ||
-        cmHasLiteralPrefix(absDestS, "/usr/lib64/") ||
-        cmHasLiteralPrefix(absDestS, "/usr/libx32/")) {
+    if (cmHasLiteralPrefix(absDestS, "/usr/lib/")) {
       // Handle "/usr move" symlinks created by some Linux distros.
       /* clang-format off */
       os <<
